config.json

```diff
@@ -36,7 +37,7 @@
 
 	"google_isenabled": true,
 		"googlehome_isenabled": true,
-		"googlehomeautomations": ["Automation 1", "Automation 2", "Automation 3"],
+		"googlehomeautomations": ["turn off the tv", "turn on the tv"],
```

Comment:
Maybe reset these back to ["Automation 1", "Automation 2", "Automation 3"]

Reply:
done.